app/src/main/java/com/android/mycax/floatingvolume/services/FloatingVolumeService.java

```diff
@@ -418,6 +423,8 @@ else if (audioVolumeObserver.equals(mAudioVolumeObserverAlarm))
             options.floatingViewY = height - (height / 2);
         }
 
+        options.moveDirection=FloatingViewManager.MOVE_DIRECTION_NONE;
```

Comment:
I do not think that's necessary, as in FloatingView documentation [https://github.com/recruit-lifestyle/FloatingView/blob/master/README.md](url) FloatingViewManager.MOVE_DIRECTION_NONE is just the behaviour of the floating head, we want it to snap to edge of the display when released from touch, with this is stays where ever we left it. Also I do not think the issue is present when running the app in debug mode. Try this signed apk (compiled with only your changes) and see if it still works. [https://drive.google.com/open?id=1Offiyin8A6SqpkLGdAY8mSYRw13mMFeA](url)